Sources/Controllers/MessagesViewController.swift

```diff
@@ -56,6 +56,14 @@ UICollectionViewDelegateFlowLayout, UICollectionViewDataSource {
     /// The default value of this property is `false`.
     open var maintainPositionOnKeyboardFrameChanged: Bool = false
 
+    /// Display the date of message by swiping left.
+    /// The default value of this property is `false`.
+    open var displayTimeBySwipingLeft: Bool = false {
+        didSet {
+            addPanGesture()
```

Comment:
didSet should handle both true and false - when set to true, add the panGesture if not added already, when set false, remove the panGesture